app/charts/column/columns-stacked-state.tsx

```diff
@@ -411,7 +411,7 @@ const useColumnsStackedState = ({
           color: colors(getSegment(datum)) as string,
         },
         values: sortedTooltipValues.map((td) => ({
-          label: getSegment(td),
+          label: `${getSegment(td)}`,
```

Comment:
`getSegment` is supposed to only return strings, so I think there is no need to change that?

Reply:
Yes ! Thanks